### PR TITLE
Expand the CHANNELS file

### PR DESCRIPTION
**Pull request by thebeanogamer**
Add catagories and voice channels to the `CHANNELS.md` file.

Please double check this, I am about to fall asleep, so it may be a mess.

### CODE REVIEW

**Review by teamshortcut — APPROVED** [review]
Looks great.

File: CHANNELS.md [hunk 1]
```diff
@@ -1,32 +1,89 @@
...
+## CyberStart
+Channel | Type | Purpose | Role Permissions
+--- | --- | --- | ---
+#cyberstart | :speech_balloon: | General discussion of CyberStart programmes | @everyone can read/write
+
+## 100 Percent
+Channel | Type | Purpose | Role Permissions
+--- | --- | --- | ---
+#100-percent | :speech_balloon: | Can be used to safely discuss challenge spoilers among people who have completed 100% of CyberStart Game | @everyone denied, @100 Percent can read/write
+100 percent | :microphone: | Corresponding voice channel for #100-percent | @everyone denied, @100 Percent can read/write
```
Comment (picapi) [inline CHANNELS.md]: Small tweak to language for consistency
```suggestion
100 percent | :microphone: | Corresponding voice channel for #100-percent | @everyone denied, @100 Percent can join and talk
```

File: CHANNELS.md [hunk 2]
```diff
@@ -1,32 +1,89 @@
...
+#programming | :speech_balloon: | Discussion and help with programming | @everyone can read/write
+#linguistics | :speech_balloon: | Discussion of linguistics - leftover from April Fool's that was popular enough to leave, at least for the time being | @everyone can read/write
+
+## CyberStart Elite
+Channel | Type | Purpose | Role Permissions
+--- | --- | --- | ---
+#elite-2018 | :speech_balloon: | Discussion of CyberStart Elite 2018, particularly to keep spoiler-y discussions separate | @everyone denied, @Veteran 2018 can read/write
+#cloud-ctf | :speech_balloon: | Discussion of the CyberStart Cloud CTF Beta | @everyone denied, @Cloud CTF can read/write
+Cloud CTF | :microphone: | Corresponding voice channel for #cloud-ctf | @everyone denied, @Cloud CTF can talk
+CyberStart Elite | :microphone: | Corresponding voice channel for #elite-2018 | @everyone denied, @Veteran 2018 can read/write
```
Comment (picapi) [inline CHANNELS.md]:
```suggestion
CyberStart Elite | :microphone: | Corresponding voice channel for #elite-2018 | @everyone denied, @Veteran 2018 can join and talk
```

**Review by picapi — COMMENTED** [review]
Looks awesome, just suggesting a couple of wording changes for accuracy

**Review by lightspeedana — APPROVED** [review]

**Pull request merged**